so-elastalert/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ghcr.io/jertel/elastalert2/elastalert2:2.9.0
+FROM jertel/elastalert2
```

Comment:
Why did you move this back to DockerHub? It sourced from GHCR to avoid the throttling imposed by Docker Corp. Also, we should pin it to a specific release, otherwise it will auto-pull in newer Elastalert2 releases which could contain breaking changes.